components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/messaging_reference.adoc

```diff
@@ -0,0 +1,446 @@
+[id="messaging_reference"]
+= Messaging reference
+:description: Comprehensive function and event reference for Starknet messaging.
+
+The Starknet messaging smart contracts include functions and events that you use to implement various flows in a dApp.
```

Comment:
this needs to be rewritten to match the rest of the docs style

Reply:
Responded in #1209.